### PR TITLE
Update time tracking section.

### PR DESCRIPTION
**Pull request by diazrenata**
## Content changes/additions

Updates and adds text to the time tracking section reflecting the Toggl discussions we had at the January 2024 retreat. 

## Technical changes/additions

None

If you are a first time contributor remember to add yourself to CITATION.cff and to the author section in \_quarto.yml!

N/A

### CODE REVIEW

File: project-management.qmd [hunk 1]
```diff
@@ -28,17 +28,20 @@ Our group uses the following hierarchy of meetings, from brief daily status repo
 
 ## Time Tracking
 
-Every individual in the group contributes to multiple projects.
+We use [Toggl](https://toggl.com) to track the time each group member spends working on each of the (many) projects that we all contribute to.
```
Comment (Aariq) [inline project-management.qmd:31]:
```suggestion
We use [Toggl](https://track.toggl.com) to track the time each group member spends working on each of the (many) projects that we all contribute to.
```
Toggl make another product in addition to the time tracker, this URL goes directly to the tracker

File: project-management.qmd [hunk 2]
```diff
@@ -28,17 +28,20 @@ Our group uses the following hierarchy of meetings, from brief daily status repo
...
 
-With the launch of the Data Science Incubator, we found it prudent to track time allocated to each project.
-The goal is to be fair to our collaborators and funders by quantifying our time spent and making sure that we are fairly allocating our efforts across projects.
-This will also be used to budget for the time and costs of future projects.
+This began with the Data Science Incubator, when it became prudent to track the time spent working on each project.
+We now use it in order to be fair to our collaborators and funders by quantifying the time spent on projects, and making sure that we are allocating effort appropriately across projects.
+We also use it to populate the Group Effort Spreadsheet, which we use to communicate our activities to stakeholders within the university.
```
Comment (Aariq) [inline project-management.qmd:35]:
```suggestion
We also use it to populate the [Group Effort Spreadsheet](impact.qmd#where-to-track-enagements), which we use to communicate our activities to stakeholders within the university.
```

File: project-management.qmd [hunk 3]
```diff
@@ -28,17 +28,20 @@ Our group uses the following hierarchy of meetings, from brief daily status repo
...
 
-The goal is not to account for 100% of each person's time.
-The goal is to be able to confidently and correctly report that we have spent a particular number of days or fraction of time on a project.
```
Comment (Aariq) [inline project-management.qmd:38]: I kind of liked this reminder that we are not expected to account for 100% of our time and that it is not being used to "make sure we are working" or anything like that

Comment (diazrenata) [reply project-management.qmd:38]: Done via 5f8e43dec19a036f40fcfe51173d50b2284a1091

**Pull request merged**